### ISSUE TITLE
Links to document open the Notion file not the MD file.

### ISSUE THREAD

**Issue by JenniBeadle** [sillsdev/docu-notion#18]
**Describe the bug**
What is the best way to add links?  
If I add the link in Notion then when I click on the link it takes me to the **Notion** file not the **webpage**. 
I have many links in my project which worked well in raw MD files but my conversion process (Word add-in) has hard-coded them to a file on my c: drive so I need to replace them.   

** Reproduction Steps **
Goto https://sillsdev.github.io/paratext-manual/Overview/
Click on link to the right of the chapter name (e.g. 1.Intro)
The notion page opens.

URL of a source Notion page that, when docu-notion accesses or converts it, shows the problem:
https://jennibeadle.notion.site/00-Overview-3ce84f9ca3264aed8532489c0574f95c
(see links in Introduction)

At least one of these:
1) URL of a docusuarus site showing the resulting page
https://sillsdev.github.io/paratext-manual/Overview/
(see links in Introduction)

**Expected behavior**
When I click on the link on a page I expect to go to the appropriate page on the website.
So if I clicked on 1.Intro I expect to go to 
https://sillsdev.github.io/paratext-manual/288f2a75-e689-4ec7-98f8-17f982c15e17/
not the Notion page.

**Screenshots**

Choose the link
<img width="283" alt="image" src="https://user-images.githubusercontent.com/108683165/197030741-e23041b0-8696-4123-8c27-8c95a1caa011.png">

Expected result
<img width="541" alt="image" src="https://user-images.githubusercontent.com/108683165/197030936-58e41001-f469-48c4-9d95-ca17ebe6119b.png">

Actual result
<img width="590" alt="image" src="https://user-images.githubusercontent.com/108683165/197031066-4b3bc10f-7163-417a-aefb-5f789a764731.png">

**Additional context**

**Comment by andrew-polk** [comment]
@JenniBeadle 
I'm not sure if I understand correctly. 
But I think you just need to change the way the link is defined in Notion.

Right now, it seems it has a full path to something like `https://jennibeadle.notion.site/Paratext-Course-Introduction-288f2a75e6894ec798f817f982c15e17` (I can't tell exactly how it is defined because I don't have edit access).

The typical way to do internal linking in Notion, it doesn't have the full path to the Notion site but rather the relative path from the root. That is typically done by this Notion UI:
![image](https://user-images.githubusercontent.com/5847219/205348299-b8133584-4533-40dd-a4cb-14cfa52be95c.png)

If you're trying to find a way to automate that, I **think** you may just need to replace the full link with the hash from the end of it.
So **maybe** 
`https://jennibeadle.notion.site/Paratext-Course-Introduction-288f2a75e6894ec798f817f982c15e17`
becomes
`/288f2a75e6894ec798f817f982c15e17`
but you would have to see if that works.

Please let me know if that answers your question.

**Comment by JenniBeadle** [comment]
Well yes, that was my question - how should I define the links? What do I change it to?

As I tried to explain above, and you have seen, the import process has added a full path for the links. So obviously I need to change them. Notion doesn't want to allow me to edit them so it looks like it will take laborious manual changes (around 500 or so) adding them using the UI. But  I can't get that to work either. 

I had another go tonight. I deleted some imported links and used the Notion interface to add a link. Now I get a Page not found error.  I guess that is progress.

Now that I understand more about IDs in Notion, I am wondering if it is had anything to do with the fact that I have tried to avoid those longs codes and instead have used the short meaningful filenames I have 
 in my original Docusaurus site.  

**I have given you edit access so you can play with it**.

BTW in the meantime I have done considerable editing to my original Docusaurus site so I will probably have to start again and reimport the data (127 files with 150+ images and 500+ links). Without a replace function in Notion, it will be important to get them right.

**Comment by andrew-polk** [comment]
@JenniBeadle 
I'm not sure how else I can help with this right now. 
I'm going to close the issue, but if you need something further, you can comment here again or open a new issue.

**Comment by JenniBeadle** [comment]
Sorry I feel like this has just taken me around in a circle, so I am no further ahead.  So please let me try again. 

Hopefully it is obvious, but **my goal** is to add links to my Notion file that work in the Docusaurus website and go to the correct page on that site. But nothing I do works.

**What we  have done so far**
I stated, and you proved, that using the imported links will never work as they are hard-coded.
Notion won't let me modify the link, but it will let me redefine them.
You stated that I need to add the links with the UI.
But that's what I tried and failed before I submitted this issue. As I said it goes to Notion not Docusaurus site.

**What I have tried today**
So I tried again. With a **link to page** /link and an **inline link** [[link
<img width="377" alt="image" src="https://user-images.githubusercontent.com/108683165/205763474-3341898d-33fe-4430-aa5b-7498a129b9bf.png">

**But something is going terribly wrong with links made from the UI as well.**
Instead of going to the Docusaurus site it still goes to my Notion database. I tried logging out of Notion first in case it prioritised an open Notion database. But then it gave me an extra link to go to an external Notion site.
<img width="314" alt="image" src="https://user-images.githubusercontent.com/108683165/205760057-b1004f9f-01e5-4a53-a6b9-b6a0a182d5f4.png">

**Expected links**
On my other site using raw MD files, I have it separated into two - Paratext Manual and Video Summaries.
<img width="199" alt="image" src="https://user-images.githubusercontent.com/108683165/205761069-76dac921-cf44-473b-bdb8-b65319d4d2bb.png">  
Docu-notion has succeeded in creating the same structure 👍 
<img width="220" alt="image" src="https://user-images.githubusercontent.com/108683165/205761164-cf61f929-73ac-4fce-959c-680ef2cfd04e.png">
But the actual links create are not following that structure.

**URLs in my working site**
All my links to the **manual** start with **https://manual.paratext.org/Training-Manual/...** (e.g. 
https://manual.paratext.org/Training-Manual/Stage-1/OD) 
All links to the **video summaries** start with **https://manual.paratext.org/Video-summaries/**... (e.g. https://manual.paratext.org/Video-summaries/Introduction/0.2.Navigation/0.2.1c)

**URLs in docu-notion**
In the docu-notion site the URLs are very different. 

**A link to a page** starts as expected 
<img width="314" alt="image" src="https://user-images.githubusercontent.com/108683165/205761563-9d43eb73-cd52-49f3-82ba-035bbc98401b.png">
then redirects to 
<img width="449" alt="image" src="https://user-images.githubusercontent.com/108683165/205761459-7c545d08-1dd0-4fa8-8d95-0d2e5331395a.png">
but gives me a **Page Not Found** error for page
https://sillsdev.github.io/paratext-manual/Video-summaries/Introduction/0.2-Windows-and-Navigation/0.2.1c/

As mentioned above the link I added as **inline link** either takes me **to the Notion page** if I am logged in or takes me to a link to load an external version, but still Notion not Docusaurus.

**What I have discovered**
I looked for the actual link on the deployed Docusaurus site that I want the link to go to it is:
https://sillsdev.github.io/paratext-manual/0.2.1c/
That is, without the **Video-summaries/Introduction/0.2-Windows-and-Navigation/** levels.
  
I am beginning to think that this **is a bug in docu-notion**?? If so, then I guess I should be adding it as a bug report, not reopening a comment, but I think it is good to have the history of this issue.

It has taken me over 2 hours to get to this point. But please let me know what you want me to do, so that we can resolve this issue.

Thanks.

**Comment by andrew-polk** [comment]
I met with Jenni today to try to understand her workflow and the issue better.
In the meantime, I upgraded paratext-manual to use the latest docu-notion (and docusaurus).
When we looked at the new deployment after those changes, the internal links were working (which I was slightly hopeful for based on some of the fixes/improvements we have made recently to docu-notion).

So I'm closing this again, hopeful that will get Jenni going.
